File: backend/Domain/Entities/task_type.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class task_type : BaseLogDomain
    {
        public int id { get; set; }
		public string name { get; set; }
		public string code { get; set; }
		public string description { get; set; }
		public bool? is_for_task { get; set; }
		public bool? is_for_subtask { get; set; }
		public string icon_color { get; set; }
		public int? svg_icon_id { get; set; }
		
    }
}

File: frontend/src/constants/task_type.ts
import { Dayjs } from "dayjs";

export type task_type = {
  
  id: number;
  name: string;
  code: string;
  description: string;
  is_for_task: boolean;
  is_for_subtask: boolean;
  icon_color: string;
  svg_icon_id: number;
};
